lib/identity/account.rb

```diff
@@ -37,19 +37,17 @@ class Account < Sinatra::Base
       post "/accept/ok" do
         begin
           api = HerokuAPI.new(ip: request.ip, request_ids: request_ids,
-            version: 2)
-          res = api.post(path: "/invitation2/save", expects: 200,
-            body: URI.encode_www_form({
-              "id"                          => params[:id],
-              "token"                       => params[:token],
-              "user[password]"              => params[:password],
-              "user[password_confirmation]" => params[:password_confirmation],
-              "user[receive_newsletter]"    => params[:receive_newsletter],
+            version: 3)
+          res = api.patch(path: "/invitations/#{params[:token]}", expects: 200,
+            body: MultiJson.encode({
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Redundant curly braces around a hash parameter.